force-app/main/default/lwc/hot_recordFiles/hot_recordFiles.js

```diff
@@ -0,0 +1,30 @@
+import { LightningElement, api, wire } from 'lwc';
+import getContentDocuments from '@salesforce/apex/HOT_RelatedFilesListController.getContentDocuments';
+import createBaseUrlLink from '@salesforce/apex/HOT_RelatedFilesListController.createBaseUrlLink';
+import { setDefaultValue } from 'c/componentHelperClass';
+
+export default class hot_recordFiles extends LightningElement {
+    @api recordId;
+    @api title;
+
+    contentDocuments = [];
+    contentDocumentsEmpty = true;
+
+    get defaultTitle() {
+        return setDefaultValue(this.title, 'Vedlegg');
+    }
+
+    @wire(getContentDocuments, { recordId: '$recordId' })
+    async wiredgetContentDocuments(result) {
```

Comment:
Hvorfor er den async?

Reply:
Fordi getContentDocuments funksjon returnerer en promise som noen ganger ikke er resolved før det vises på skjerm, så derfor venter jeg på at getBaseUrlLink har returnert en verdi før jeg viser det. Spesielt på "rediger"-skjermen var dette et problem, som nå funker fint for alle.